waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -46,31 +47,36 @@ migrateDev projectDir migrateArgs =
     scriptArgs =
       if System.Info.os == "darwin"
         then -- NOTE(martin): On MacOS, command that `script` should execute is treated as multiple arguments.
-          ["-Fq", "/dev/null"] ++ prismaMigrateCmd
+          ["-Fq", "/dev/null"] ++ buildPrismaMigrateCmd id
         else -- NOTE(martin): On Linux, command that `script` should execute is treated as one argument.
-          ["-feqc", unwords prismaMigrateCmd, "/dev/null"]
-
-    -- NOTE(martin): For this to work on Mac, filepath in the list below must be as it is now - not
-    -- wrapped in any quotes.
-    -- NOTE(martin): We do "--skip-seed" here because I just think seeding happening automatically
-    --   in some situations is too aggressive / confusing.
-    prismaMigrateCmd =
-      [ absPrismaExecutableFp projectDir,
+          ["-feqc", unwords $ buildPrismaMigrateCmd Shell.quoteArg, "/dev/null"]
+
+    -- NOTE(miho): Since we are running the Prisma command using `script` and we are doing it
+    --  in two different ways (MacOS and Linux), we have to take care of quoting the paths
+    --  differently.
+    --  * MacOS - we are passing the command as multiple arguments, so we MUST NOT quote the paths.
+    --  * Linux - we are passing the command as one argument, so we MUST quote the paths.
+    buildPrismaMigrateCmd quoteArg =
+      [ quoteArg $ absPrismaExecutableFp projectDir,
```

Comment:
```suggestion
    -- NOTE(miho): Since we are running the Prisma command using `script` and we are doing it
    --  in two different ways (MacOS and Linux), we have to take care of quoting the path arguments
    --  differently.
    --  * MacOS - we are passing the command as multiple arguments, so we MUST NOT quote the path arguments.
    --  * Linux - we are passing the command as one argument, so we MUST quote the path arguments.
    buildPrismaMigrateCmd :: (String -> String) -> [String]
    buildPrismaMigrateCmd argQuoter =
      [ argQuoter $ absPrismaExecutableFp projectDir,
```